source/extensions/tracers/opentelemetry/samplers/sampler.h

```diff
@@ -37,11 +38,18 @@ enum class Decision {
  */
 using OTelSpanKind = ::opentelemetry::proto::trace::v1::Span::SpanKind;
 
+/**
+ * @brief Open-telemetry Attribute
+ * see
+ * https://github.com/open-telemetry/opentelemetry-cpp/blob/main/api/include/opentelemetry/common/attribute_value.h
+ */
+using OTelAttribute = ::opentelemetry::common::AttributeValue;
+
 struct SamplingResult {
   /// @see Decision
   Decision decision;
   // A set of span Attributes that will also be added to the Span. Can be nullptr.
-  std::unique_ptr<const std::map<std::string, std::string>> attributes;
+  std::unique_ptr<const std::map<std::string, OTelAttribute>> attributes;
```

Comment:
nit: giving the `std::map<std::string, OTelAttributes>` a type alias like `OtelAttributes`.

Reply:
Thanks for the hint, done in 5eb178892b8ca0a23bad2e685a55d6354dcf5a9f